modules/ROOT/pages/set-dlb.adoc

```diff
@@ -0,0 +1,128 @@
+[%header%autowidth.spread]
+|===
+|Index |Input URI (Input Path) |App name |App URI (Output Path) | Protocol
+|0 |`/graphql` |`datagraph-xz082edx-1585-901xe374-d0x0` |`/graphql` |HTTPS
+|1 |`/datagraph/` |`datagraph-xz082edx-1585-901xe374-d0x0` |`/graphql` |HTTPS
+|2 |`/datagraph/` |`datagraph-xz082edx-1585-901xe374-d0x0` |`/` |HTTPS
+|3 |`/{app}/` |`{app}` | `/` |HTTPS
+|===
+
+For each index, the appropriate DLB URL is:
+
+* Index 0: `examplelb.lb.anypointdns.net/graphql`
+* Index 1: `examplelb.lb.anypointdns.net/datagraph/`
+* Index 2: `examplelb.lb.anypointdns.net/datagraph/graphql`
+* Index 3: `examplelb.lb.anypointdns.net/datagraph-xz082edx-1585-901xe374-d0x0/graphql`
+
+=== Prerequisites
+
+* You must have the xref:permissions.adoc[Admin or Operator permission] for the environment in which you want to specify a DLB URL.
+* You must be able to xref:runtime-manager::anypoint-platform-cli#authentication[authenticate to] and run commands from the Anypoint CLI.
+* You must be familiar with xref:runtime-manager::lb-mapping-rules[creating DLB mapping rules].
+* You must xref:runtime-manager::cloudhub-dedicated-load-balancer.adoc#create-and-configure-a-dedicated-load-balancer[create and configure a DLB].
+
+To complete this task, you:
+
+. Use the Anypoint CLI to gather details about your DLB and Anypoint DataGraph.
+. Use Anypoint Runtime Manager to create a mapping rule for the DLB.
+. Use the Anypoint CLI to add the DLB configuration to Anypoint DataGraph.
+
+=== Gather Details about the DLB and Anypoint DataGraph
+
+Gather the name of your DLB and the app name for Anypoint DataGraph.
+
+. Connect to the Anypoint CLI using your Anypoint Platform username and password.
+. If you did not previously set the environment, do so with the following command:
++
+`use environment <environment name>`
+. Get the name of the DLB:
++
+`cloudhub load-balancer describe <dlb name>`
++
+The expected output resembles:
++
+`Name | examplelb`
++
+. Get the CloudHub domain for the Anypoint DataGraph app name:
++
+`datagraph load-balancer-config describe`
++
+The expected value resembles:
++
+`Domain |  datagraph-xz082edx-1585-901xe374-d0x0`
++
+The DataGraph app name contains elements from your org ID and the environment ID.
+. Note the values for steps 3 and 4.
+
+=== Create a Mapping Rule for the DLB
+
+You set up the mapping rule in Runtime Manager.
+
+. From Anypoint Platform, select *Runtime Manager*.
+. From the navigation menu, select *Load Balancers*.
+. Click a load balancer name, and then click a certificate name.
++
+Ensure your load balancer has a valid, CA-signed certficate, or the browser won't connect to Anypoint DataGraph. 
+. In the *URL Mapping Rules* section, click *Add New Rule*.
+. Complete the fields with the following values:
+** *Input Path* (`inputUri`): The URI that the client requests: for example, `/{app}/`
+** *Target App* (`appName`): The domain for the Anypoint DataGraph application discovered in Step 4 of the gathering details task
```

Comment:
Generally I would refer to the task but not the specific step in the task. However, this is up to you, especially if Kim or someone else has recommended referring to the specific step.
```suggestion
** *Target App* (`appName`): The domain for the Anypoint DataGraph application discovered in the gathering details task
```

Reply:
I'm going to leave it since Kim added it